python/python/glide/async_commands/stream.py

```diff
@@ -363,6 +363,69 @@ def __init__(
         self.consumer_name = consumer_name
 
 
+class StreamClaimOptions:
+    IDLE_REDIS_API = "IDLE"
+    TIME_REDIS_API = "TIME"
+    RETRY_COUNT_REDIS_API = "RETRYCOUNT"
+    FORCE_REDIS_API = "FORCE"
+    JUST_ID_REDIS_API = "JUSTID"
+
+    def __init__(
+        self,
+        idle: Optional[int] = None,
+        idle_unix_time: Optional[int] = None,
+        retry_count: Optional[int] = None,
+        is_force: Optional[bool] = False,
+    ):
+        """
+        Options for `XCLAIM`.
+
+        Args:
+            idle (Optional[int]): Set the idle time (last time it was delivered) of the message in milliseconds. If idle
+                is not specified, an idle of 0 is assumed, that is, the time count is reset because the message now has a
+                new owner trying to process it.
+            idle_unix_time (Optional[str]): This is the same as idle but instead of a relative amount of milliseconds,
```

Comment:
```suggestion
            idle_unix_time (Optional[int]): This is the same as idle but instead of a relative amount of milliseconds,
```

Reply:
Yury, which hardware store did you get your laser eyes from, I need name, address and aisle number, please 🙏